src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -683,4 +708,134 @@ public function column_cb( $post ) {
 			<div class="locked-indicator"></div>
 		<?php
 	}
+
+	/**
+	 * Displays the pagination.
+	 *
+	 * @since TBD
+	 * @access protected
+	 *
+	 * @param string $which Equal to NULL, 'top' or 'bottom'.
+	 */
+	protected function pagination( $which ) {
+		if ( empty( $this->_pagination_args ) ) {
+			return;
+		}
+
+		$total_items     = $this->_pagination_args['total_items'];
+		$total_pages     = $this->_pagination_args['total_pages'];
+		$infinite_scroll = false;
+		if ( isset( $this->_pagination_args['infinite_scroll'] ) ) {
+			$infinite_scroll = $this->_pagination_args['infinite_scroll'];
+		}
+
+		if ( 'top' === $which && $total_pages > 1 ) {
+			$this->screen->render_screen_reader_content( 'heading_pagination' );
+		}
+
+		$output = '<span class="displaying-num">' . sprintf(
+			/* translators: %s: Number of items. */
+			_n( '%s item', '%s items', $total_items ),
+			number_format_i18n( $total_items )
+		) . '</span>';
+
+		$current              = $this->get_pagenum();
+		$removable_query_args = wp_removable_query_args();
+
+		$current_url = set_url_scheme( 'http://' . sanitize_text_field ( $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'] ) );
```

Comment:
* :no_entry_sign: Space before opening parenthesis of function call prohibited
* :no_entry_sign: Detected usage of a possibly undefined superglobal array index: $_SERVER['HTTP_HOST']. Use isset() or empty() to check the index exists before using it
* :no_entry_sign: Detected usage of a possibly undefined superglobal array index: $_SERVER['REQUEST_URI']. Use isset() or empty() to check the index exists before using it

<sub>hash: d3686e79c34023945ed22d02b7df14ae</sub>